src/selectors/entities/channels.js

```diff
@@ -222,35 +222,37 @@ export const getOtherChannels = createSelector(
 export const getChannelsByCategory = createSelector(
     getCurrentChannelId,
     getMyChannels,
+    getConfig,
     getMyPreferences,
     getTeammateNameDisplaySetting,
     (state) => state.entities.users,
-    (currentChannelId, channels, myPreferences, teammateNameDisplay, usersState) => {
+    (currentChannelId, channels, config, myPreferences, teammateNameDisplay, usersState) => {
         const allChannels = channels.map((c) => {
             const channel = {...c};
             channel.isCurrent = c.id === currentChannelId;
             return channel;
         });
 
-        return buildDisplayableChannelList(usersState, allChannels, myPreferences, teammateNameDisplay);
+        return buildDisplayableChannelList(usersState, allChannels, config, myPreferences, teammateNameDisplay);
```

Comment:
I think this is ok for now but actually we are trying to move away from this functions as they are really poor on performance